inc/customizer/site-identity.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * Additional site identity customizer options.
+ *
+ * @package Primer
+ * @since   1.4.2
+ */
+
+class Primer_Site_Identity_Options {
+
+	/**
+	 * Class constructor.
+	 */
+	public function __construct() {
+
+		if ( class_exists( 'WPaaS\Log\Components\Plugin' ) && WPaaS\Plugin::is_reseller() ) {
```

Comment:
@EvanHerman I would expose a filter and put this into the system plugin hotfix class.